src/runtime_src/core/pcie/tools/xbmgmt/xmc.cpp

```diff
@@ -600,7 +600,11 @@ static void tiTxtStreamToBin(std::istream& tiTxtStream,
         switch (line[0]) {
         case '@':
             // Address line
-            currentAddr = std::stoi(line.substr(1), NULL , 16);
+            try {
+                currentAddr = std::stoi(line.substr(1), NULL, 16);
+            } catch (...){
+                return;
```

Comment:
This is great that we are catch the exception and the original issue is addressed.

The next step is to let the user "know" what just happened.  For example, and error message similar to:
`ERROR: The format of the current address '<the given string>' isn't valid.  No action taken.`
Or something like that.